### DISCUSSION TITLE
Linux LSX Authentication with Lutris and Origin

### DISCUSSION THREAD

**Discussion by 7ritn** [R2Northstar/Northstar#211] [Q&A]
### Bug
Trying to run northstar on Linux with Lutris fails with an Authentication Error

### Steps to reproduce
  1. Install Titanfall with Lutris
  2. Drop in Northstar files into the Titanfall folder
  3. Try to launh the game

### Specifications
  - Northstar version: v.1.5.1
  - Platform: Origin/Lutris

### Troubleshooting
I tried all the steps layed out in the troubleshooting section of the wiki:

> First and foremost, double check that you are logged in on Origin. Titanfall will not run if you are not connected to EA servers first (and neither will Titanfall + Northstar).

Yes I'm logged in and I can launch vanilla Titanfall without any problem

> Add the northstar command line option to your launcher
> Run the game with Origin/Steam instead of starting NorthstarLauncher.exe **(important)**

**When I do this, the game launches vanilla, which is weird**

> Sign out and exit Origin, then start Northstar. It will prompt you for a login, hopefully fixing it if nothing else worked

Did not work as well

> Start normal Titanfall 2 and then Northstar (ONLY WORKS SOMETIMES)

When I try to do this I get a memory violation not 0x0..000. Don't know if this is interesting, prob not.

I'd really like to play Northstar on Linux, on Windows everything works as expected.

**Comment by abarichello** [comment]
I'm also using Origin/Lutris on Linux, when I get the auth errors in-game I have to kill all the wine processes and re-launch TF|2 to get it working again. Try that
![image](https://user-images.githubusercontent.com/16687318/156848556-c95f6132-2784-4b7c-9f18-775a68c0fc0c.png)

**Reply by 7ritn** [reply]
So I get this Auth Error, before even getting into the main menu. Anyway I tried your solution, but no success. After the Auth Error, I killed all Wine processes and started Northstar from Lutris again. After this I get a different error: "Origin could not be initialized: The Origin Desktop Application is not loaded.". However Origin is running in the background. Re-launching again I get the Auth Error.
I also have Titanfall on my root file system so it can't be a NTFS problem. I tried lutris-fshack-7.1-x86_64 and also the latest proton GE.

My system specs are as follows:

> Operating System: Manjaro Linux
> KDE Plasma Version: 5.24.2
> KDE Frameworks Version: 5.91.0
> Qt Version: 5.15.2
> Kernel Version: 5.15.25-1-MANJARO (64-bit)
> Graphics Platform: X11
> Processors: 8 × Intel® Core™ i7-4790K CPU @ 4.00GHz
> Memory: 15.6 GiB of RAM
> Graphics Processor: AMD Radeon RX 6700 XT

I could potentialy try running it with the EA App, but for this I'd have to download the files again, because EA App does not support finding installed games yet.

**Reply by abarichello** [reply]
Sorry, not sure how to help you. Origin is certainly the worst part when it gets to playing on Linux. If you find a solution please post here

**Reply by 7ritn** [reply]
I tried using the EA App Beta, but the results were the same, so I'm still looking for a solution. But I'm just not sure if there's anything left to try.

**Comment by j6b72** [comment]
I found [this](https://r2northstar.gitbook.io/r2northstar-wiki/using-northstar/playing-on-linux) wiki article. Replacing the default Titanfall2.exe's path with the NorthstarLauncher.exe as described in ["Steam (Proton)"](https://r2northstar.gitbook.io/r2northstar-wiki/using-northstar/playing-on-linux#steam-proton) and then starting the game normally (in my case via Origin) worked.
